test/Vagrantfile

```diff
@@ -6,13 +6,13 @@ $JOB_NAME = ENV['JOB_BASE_NAME'] || "LOCAL"
 $K8S_VERSION = ENV['K8S_VERSION'] || "1.8"
 $K8S_NODES = (ENV['K8S_NODES'] || "2").to_i
 
-$SERVER_BOX= "cilium/ginkgo"
-$SERVER_VERSION="0.0.4"
+$SERVER_BOX= "cilium/ubuntu"
```

Comment:
Is this the image from `packer-ci-build`?

Reply:
Yes